07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,45 @@
+body {
+    width: 850px;
+    margin: 0 auto;
+}
+header {
+  background-image: url('../img/nasa-background.jpg');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 450px;
+  text-align: center;
+}
+.title {
+  font-size: 60px;
+  color: #fff;
+  width: auto;
+  line-height: 450px;
+}
+section {
+  height: 324px;
+  background: lightgrey;
+  margin-top: 10px;
+}
+div {
+  float: left;
+  width: 250px;
+  margin-left: 20px;
+  vertical-align: top;
+  display: inline-block;
+
+}
+div:nth-child(2){
+  margin: 0 20px;
+}
+label {
```

Comment:
if `label` and `input` have the same `declaration`, how can you combine the selectors to reduce redundancy?